src/models/country/id.yaml

```
@@ -1,9 +1,12 @@
 type: object
 required:
   - id
-description: "The identifier for the country"
 properties:
   id:
+    description: |
+      The unique identifier for the country. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)